### PR TITLE
Add Rizk to Committee

### PR DESCRIPTION
**Pull request by HPrivakos**

### CODE REVIEW

File: src/entities/Teams.ts [hunk 1]
```diff
@@ -74,14 +74,19 @@ export const DAOCommittee = new Committee(
...
+      avatar: 'https://peer.decentraland.org/content/contents/bafkreidnd3mgh25roc6g4scwkidkbo5pdr2w3lqpjecrnadx6dns72ojdy'
     }
   ]
 )
 
 export const CurationCommittee = new Committee(
   'Wearable Curation Committee',
   'Responsible for reviewing new wearable submissions ensuring they are glitch-free and compliant with the design guidelines. They also rise warnings about IP infringement and violent content.',
-  15,
+  100,
```
Comment (andyesp) [inline src/entities/Teams.ts:89]: Could you move this change to another PR so we can merge adding Rizk first?

Comment (andyesp) [inline src/entities/Teams.ts:89]: Could you move this change to another PR so we can merge adding Rizk first?

**Review by andyesp — CHANGES_REQUESTED** [review]

File: src/entities/Teams.ts (hunk 1 above)
Comment (HPrivakos) [reply src/entities/Teams.ts:89]: This limit is arbitrary, I don't think there is any discussion to have on increasing this number, but sure

**Review by ncomerci — APPROVED** [review]

**Review by 1emu — APPROVED** [review]
